github-collaboration/github-collaboration.md

```diff
@@ -13,8 +13,19 @@ About 1 hour
 - Terminal or Command Prompt
 
 ### Motivation
-In the industry, apps are created by teams of developers rather than individual people. Proper use of GitHub for collaboration minimizes code conflicts and helps teams work together.
+- In the industry, apps are created by teams of developers rather than individual people. Proper use of GitHub for collaboration minimizes code conflicts and helps teams work together. 
+- At the heart of collaboration, developers use branches when working on new features or patching a bug. This makes it easier to deal with multiple contexts while developing in a professional manner
```

Comment:
```suggestion
- At the heart of collaboration, developers use branches when working on new features or patching a bug. This makes it easier to deal with multiple contexts while developing in a professional manner.
```
This looks great so far!  Can you elaborate on what you mean by ' deal with multiple contexts' ?

Reply:
Oh sorry just saw this, I have explained this here: https://github.com/Techtonica/curriculum/pull/1046 

I deleted my fork when I saw the other PR was merged, I forgot there was an open one here too. Kindly review there while I close this